examples/employee-app/src/main/java/com/example/helidon/employee/Employee.java

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2019 Oracle and/or its affiliates. All rights reserved. */
+package com.example.helidon.employee;
+
+import java.util.UUID;
+
+public class Employee {
+
+    private String id;

```

Comment:
Consider making all the fields final and forcing the use of the factory method `of`.
See an example of doing that here: https://gist.github.com/romain-grecourt/d31339e21e9cb541b1eb121e526d25d9

Reply:
This should be fixed now.